### PR TITLE
greek translation specifics

### PR DESCRIPTION
**Pull request by gemantzu**

### CODE REVIEW

File: gr/lessons/specifics/ecto.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+      supervisor(ExampleApp.Repo, [])
+    ]
+
+    opts = [strategy: :one_for_one, name: ExampleApp.Supervisor]
+    Supervisor.start_link(children, opts)
+  end
+end
+```
+
+Για περισσότερες πληροφορίες στους επιτηρητές ελέγξτε το μάθημα [Επιτηρητές OTP](/lessons/advanced/otp-supervisors).
```
Comment (nscyclone) [inline gr/lessons/specifics/ecto.md]: I think that this link should lead to the Greek lesson, not the English one.
`[Επιτηρητές OTP](/lessons/advanced/otp-supervisors)` → `[Επιτηρητές OTP](/gr/lessons/advanced/otp-supervisors)`.

File: gr/lessons/specifics/mnesia.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+Δεν είναι πάντα ξεκάθαρο το πότε θα χρησιμοποιήσουμε ένα κομμάτι τεχνολογίας.  Αν μπορείτε να απαντήσετε 'ναι' σε κάποια από τις παρακάτω ερωτήσεις, τότε αυτή είναι μια καλή ένδειξη για τη χρήση του Mnesia αντί των ETS ή DETS.
+
+  - Χρειάζεται να γυρίσω πίσω μεταφορές;
+  - Χρειάζομαι ένα εύκολο στη χρήση συντακτικό για την ανάγνωση και εγγραφή δεδομένων;
+  - Πρέπει να αποθηκεύσω δεδομένα σε πάνω από ένα κόμβο;
+  - Χρειάζομαι μια επιλογή στο που να αποθηκεύσω πληροφορίες (στη RAM ή στο δίσκο);
+
+## Σχήμα
+
+Καθώς το Mnesia είναι μέρος πυρήνα της Erlang, όχι της Elixir, πρέπει να χρησιμοποιήσουμε το συντακτικό με την άνω κάτω τελεία για να έχουμε πρόσβαση σε αυτό (δείτε το μάθημα: [Αλληλεπίδραση με την Erlang](/lessons/advanced/erlang/)) ως εξής:
```
Comment (nscyclone) [inline gr/lessons/specifics/mnesia.md]: Same issue here:
`[Αλληλεπίδραση με την Erlang](/lessons/advanced/erlang/)` → `[Αλληλεπίδραση με την Erlang](/gr/lessons/advanced/erlang/)`.

**Comment by nscyclone** [comment]
Thanks, @gemantzu!
I've left some notes about links. Once they're fixed, this is ready to merge.

File: gr/lessons/specifics/ecto.md (hunk 1 above)
Comment (gemantzu) [inline gr/lessons/specifics/ecto.md]: I really thought that it used current folder, not root one.  This issue might have crept all over my translations.  I will scan through them to see if it is correct or not.

**Comment by gemantzu** [comment]
They should be ok now.  I will scan through the guides to see if this has made it through to any other lessons.

**Comment by nscyclone** [comment]
Thanks, @gemantzu!
I suppose that all the similar shortcomings are already fixed in #634.

**Comment by gemantzu** [comment]
Ow you already checked it? ok then. :) Thanks @nscyclone !

**Pull request merged**